template/composer.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "zendframework/<package_name>",
+    "description": "<package_description, the same like the repository description>",
+    "homepage": "https://docs.zendframework.com/<package_name>/",
+    "license": "BSD-3-Clause",
+    "keywords": [
+        "zend",
+        "zendframework",
+        "<package_name>",
+        "<individual keywords for current package>"
+    ],
+    "support": {
+        "docs": "https://docs.zendframework.com/<package_name>/",
+        "issues": "https://github.com/zendframework/<package_name>/issues",
+        "source": "https://github.com/zendframework/<package_name>",
+        "rss": "https://github.com/zendframework/<package_name>/releases.atom",
```

Comment:
I find it funny that we use an Atom feed for the RSS item. :smile: That said, it's the only appropriate location.